scipy/stats/anova.py

```diff
@@ -0,0 +1,268 @@
+from types import SimpleNamespace
+from collections import namedtuple
+import numpy as np
+from ..special import stdtrit, fdtrc
+
+
+__all__ = [
+    'AnovaOnewayResult',
+    'anova_oneway',
+    'anova_oneway_from_labels_values',
+]
+
+
+ConfidenceInterval = namedtuple('ConfidenceInterval', ['low', 'high'])
```

Comment:
Given that we should be dropping 3.6 for the next release, we should be able to use dataclasses now, which I would recommend over `namedtuple`. If you want to stick to named tuples, I would recommend the newer `typing.NamedTuple` syntax:

https://docs.python.org/3/library/typing.html#typing.NamedTuple